### PR TITLE
Update gitignore

### PR DESCRIPTION
**Pull request by oshoval**
Signed-off-by: Or Shoval <oshoval@redhat.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,26 +1,2 @@
...
-
-!vendor/**/zz_generated.*
-
-# editor and IDE paraphernalia
-.idea
-*.swp
-*.swo
-*~
+_kubevirtci/
+build/
```
Comment (AlonaKaplan) [inline .gitignore:2]: Where do we create build dir?

File: .gitignore [hunk 2]
```diff
@@ -1,26 +1,2 @@
-
-# Binaries for programs and plugins
```
Comment (AlonaKaplan) [inline .gitignore:2]: Why all the content of the file was removed? It was originally auto generated and contains patterns that is useful to ignore.

Comment (oshoval) [reply .gitignore:2]: We don't need to ignore what doesn't exists
You add only what you plan to ignore in the PR that creates new stuff that need to be ignored
This is one of the drawbacks of using generator
See please all the other repos, the git ignore should be minimal
As long as git status is clean, the less the merrier

File: .gitignore (hunk 1 above)
Comment (oshoval) [reply .gitignore:2]: See please the makefile, it installs go in order to run vet / tests etc

**Review by AlonaKaplan — APPROVED** [review]

**Pull request merged**